A4_3.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+int+push;
+
+public class A4_3 {
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		Scanner scan = new Scanner(System.in);
+		System.out.print("Enter a number greater than 0 (less than 1 to quit): ");
+		
+		int num = scan.nextInt();
+		int sum = 0;
+		int square;
+		
+		if(num > 1 || num == 1) {

```

Comment:
```suggestion
		if(num >= 1) {

```

Reply:
if you want it to loop, do:
```suggestion
		while(num >=1) {

```

Reply:
ohh! 
so when do we exactly use the do-while loop? would this be a good time to use the do-while loop? just wondering